docs/user_guide/datasets/boston_housing_data.rst

```diff
@@ -447,3 +446,16 @@ you pause about using it in the future.
   
   .. [#11] Kinmberlé Crenshaw, Mapping the margins: Intersectionality, identity politics, and violence against women of color, 
       Stanford Law Review, 1993, 43(6), 1241-1299.
+
+.. _hospital_readmissions_dataset:
```

Comment:
At first I briefly introduced the dataset in the user guide, but moved it as I did because @romanlutz said  
>I'm not opposed to either one of these, but I'm not a fan of introducing the dataset in between the mitigation subsections.

in [this comment](https://github.com/fairlearn/fairlearn/pull/997#discussion_r786936129).

I agree that it does not make a lot of sense to have this dataset described in the boston housing dataset file. Shall I create a new file *hospital_readmissions_dataset.rst* , and create a separate PR with the purpose of providing information about this dataset similar to the way this is done in *dataset_x.rst*, for someone else to finish (as also suggested by roman in the comment just mentioned)?

Reply:
That seems reasonable. Seems related to #1086, or is that yet another dataset?

Reply:
That is the same dataset, so I would encourage you not to write it again :).

Reply:
Oh great! I'm glad we figured that part out. Thanks @rensoostenbach for confirming!